frontend/components/about/index.ts

```diff
@@ -0,0 +1,39 @@
+export type values = {
```

Comment:
Типы нужно называть с большой буквы и, желательно, так, чтобы имя несло в себе информацию, у меня с этим тоже проблемы, так что если придумаешь название лучше - ипользуй его
```suggestion
export type AboutValuesData = {
```